docs/user-guide/features.rst

```diff
@@ -33,6 +33,38 @@ Administrators can also :ref:`upload <create-errata-recipe>` their own errata to
 a repository. Please see the :doc:`recipes` documentation to learn how to
 perform these operations.
 
+Modularity
+^^^^^^^^^^
+
+Pulp supports following modularity_ repository mirroring use cases:
+
+* synchronization of the modularity metadata content, the
+  ``repodata/*modules.yaml.gz`` file, with either the repository immediate or
+  on-demand synchronization
+
+* publication of the modularity metadata with the repository publication
+
+* copy of the modularity metadata with the repository copy
+
+* modules published thru Pulp are consumable by the ``dnf`` client
+
+More advanced use cases are planned, especially those following from the ability
+of Pulp processing the modularity metadata file.
+
+.. Note::
+
+  Filtering content types subsets might require one to explicitly include the
+  ``metafile`` in their copy filter criteria, should they want the resulting
+  content subset to include the modularity metadata.
+
+.. Note::
+
+  The metadata not being processed by Pulp it is possible for a consumer to
```

Comment:
Bad grammar here but not sure what's being said.

"For metadata not being processed by Pulp, it is ..." maybe?

Reply:
rephrasing to state that Pulp actually never processes modularity metadata